syncmaster/schemas/v1/transfers/file/base.py

```diff
@@ -80,7 +81,10 @@ def _directory_path_is_valid_path(cls, value):
 
     @field_validator("file_name_template")
     @classmethod
-    def validate_file_name_template(cls, value):
+    def validate_file_name_template(cls, value: str) -> str:
+        if not re.match(r"^[a-zA-Z0-9_.{}-]+$", value):
```

Comment:
Please use `re.compile` to build the pattern object at the top of the module or as classvar, and the use it here